tests/ci/changelog.py

```diff
@@ -288,7 +288,7 @@ def generate_description(item: PullRequest, repo: Repository) -> Optional[Descri
     # Normalize bug fixes
     if (
         re.match(
-            r".*(?i)bug\Wfix",
+            r"(?i).*bug\Wfix",
```

Comment:
does it make any difference?

Reply:
Python regex are different: https://regex101.com/r/AAiYRn/1